Dockerfile

```diff
@@ -3,9 +3,35 @@ FROM python:3.6.3-stretch
 RUN apt-get -y update \
   && apt-get -y install autoconf-archive automake g++ libtool pkg-config unzip build-essential cmake \
      libatlas-base-dev gfortran libgtk2.0-dev libavcodec-dev libavformat-dev libswscale-dev libjpeg-dev libpng-dev \
-     libtiff-dev libv4l-dev libleptonica-dev \
- && pip install numpy==1.13.3 \
- && curl -sSL https://github.com/Itseez/opencv/archive/3.3.0.zip -o opencv3.zip \
+     libtiff-dev libv4l-dev libleptonica-dev openjdk-8-jdk openjdk-8-jre-headless ca-certificates-java \
+     clang-format-3.8 libcurl4-openssl-dev libtool python-dev python-setuptools python-virtualenv \
+     python3-dev python3-setuptools zlib1g-dev \
+  && echo "deb http://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list \
```

Comment:
if you pop these to the top before your apt-get update up there, then you can potentially just install all the things with one apt-get update / install instead of 2? Or are there other packages on the bazel repo that cause conflicts?